src/data/requestFiltering.js

```diff
@@ -82,6 +98,17 @@ export function populateRequestFilterFrom(queryStringData, me, events) {
   }
 }
 
+function matchByRegion (location, requestLocation) {
+  if (!location || !requestLocation) return true
+  if (location.sublocality || location.locality || location.county) {
+    return isNear(location, requestLocation)
+  } else if (location.state) {
+    return requestLocation.description.includes(location.state)  || requestLocation.description.includes(location.description)
```

Comment:
It is not guaranteed that the `state` property is included in the description. Try Chiang Mai, Thailand, for example. Will that break this conditional?

Reply:
You are correct. I made a request for 'Chiang Mai', Thailand and searching for destination 'Chiang Mai Province, Thailand' doesn't work. It seems to have a robust search by region we will need some additional property on the request.